elife/proofreader-php.sls

```diff
@@ -23,7 +23,7 @@ proofreader-php-repository:
 
     cmd.run:
         - name: |
-            composer --no-interaction install --no-suggest --classmap-authoritative --no-dev
+            composer --no-interaction update --no-suggest --classmap-authoritative --no-dev
```

Comment:
This can probably be reverted after the upgrade for speed, until we remove this state file at all due to containerization.